README.md

```diff
@@ -32,14 +38,12 @@ shared options.
 
 ## Source Code Organization
 
-The executables are in the `cmd` folder, we have one executable per tool listed.
+The executables are in the `cmd` folder. There are currently only two: `argot` and `racerg` (an experimental static data race detector).
 
 There is user documentation in the `doc` folder.
 
-The test data is in the `testdata` folder. All the Go source files used in the tests are in `testdata/src`.
-
 The library code, and most of the analysis implementations, is in the `analysis` folder. The main entry points are in
-the `load_progam.go` file for loading program and `analyzers.go` to call analyzers. The rest is organized in subfolders:
+the `load_progam.go` file for loading the program and `analyzers.go` to call analyzers. The rest is organized in subfolders:
```

Comment:
There's a typo here (not introduced by your changes) in `load_progam.go`

Reply:
Where is the typo?

Reply:
It should be `load_program.go` (missing `r`)